node/views/skeleton.hbs

```diff
@@ -1,26 +1,33 @@
-<!doctype html>
-<meta charset="utf-8">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link rel="stylesheet" href="/static/klaus.css">
-<link rel=icon type="image/png" href="/static/favicon.png">
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+{{!-- <link rel="stylesheet" href="/static/klaus.css" /> --}}
+<link rel="stylesheet" href="/static/output.css" />
+<link rel="icon" type="image/png" href="/static/favicon.png" />
 {{#if meta.base_href}}
-  <base href="{{ meta.base_href }}">
+<base href="{{ meta.base_href }}" />
```

Comment:
prettier does not ident lines inside of handlebars logic, like this line here? ^

Reply:
I ended up not using Prettier, support seems bad: https://github.com/prettier/prettier/issues/5340

Reply:
Oh ok so you're rewriting manually?

Reply:
I need to investigate a bit and will find a way to make it work